test/is.test

```diff
@@ -2694,219 +2724,255 @@ bmZvIDEgMCBSCi9Sb290IDEzNiAwIFIKPj4Kc3RhcnR4cmVmCjExODgxMQolJUVPRgo=}
+        qc::is uri #anchor1
     } -result 1
 
-    test is_uri-1.13 {is_uri relative} -setup {} -body {
-        qc::is_uri_valid  "./../../test.pdf#anchor1?a=1&b=2"
+    test is-uri-1.13 {is uri relative} -setup {} -body {
+        qc::is uri  "./../../test.pdf#anchor1?a=1&b=2"
     } -result 1
 
-    test is_uri-1.14 {is_uri scheme relative} -setup {} -body {
-        qc::is_uri_valid "//test.com/test/test.pdf#anchor1?a=1&b=2"
+    test is-uri-1.14 {is uri scheme relative} -setup {} -body {
+        qc::is uri "//test.com/test/test.pdf#anchor1?a=1&b=2"
     } -result 1
 
-    test is_uri-1.15 {is_uri ipv6} -setup {} -body {
-        qc::is_uri_valid {http://[FE80:0000:0000:0000:0202:B3FF:FE1E:8329]}
+    test is-uri-1.15 {is uri ipv6} -setup {} -body {
+        qc::is uri {http://[FE80:0000:0000:0000:0202:B3FF:FE1E:8329]}
     } -result 1
 
-    test is_uri-1.16 {is_uri collapsed ipv6} -setup {} -body {
-        qc::is_uri_valid {http://[::2001:db8:0:1]:80}
+    test is-uri-1.16 {is uri collapsed ipv6} -setup {} -body {
+        qc::is uri {http://[::2001:db8:0:1]:80}
     } -result 1
 
-    test is_uri-1.17 {is_uri} -setup {} -body {
-        qc::is_uri_valid "urn:mpeg:mpeg7:schema:2001urn:isbn:0451450523"
+    test is-uri-1.17 {is uri} -setup {} -body {
+        qc::is uri "urn:mpeg:mpeg7:schema:2001urn:isbn:0451450523"
     } -result 1
 
-    test is_uri-1.18 {is_uri} -setup {} -body {
-        qc::is_uri_valid "sip:gateway.com;user=phone"
+    test is-uri-1.18 {is uri} -setup {} -body {
+        qc::is uri "sip:gateway.com;user=phone"
     } -result 1
 
-    test is_uri-1.19 {is_uri} -setup {} -body {
-        qc::is_uri_valid "tel:+1-816-555-1212"
+    test is-uri-1.19 {is uri} -setup {} -body {
+        qc::is uri "tel:+1-816-555-1212"
     } -result 1
 
-    test is_uri-1.20 {is_uri} -setup {} -body {
-        qc::is_uri_valid "mailto:John.Doe@example.com"
+    test is-uri-1.20 {is uri} -setup {} -body {
+        qc::is uri "mailto:John.Doe@example.com"
     } -result 1
 
-    test is_uri1.21 {is_uri invalid} -setup {} -body {
-        qc::is_uri_valid "htt$%://test.com"
+    test is-uri1.21 {is uri invalid} -setup {} -body {
+        qc::is uri "htt$%://test.com"
     } -result 0
 
-    test is_uri1.22 {is_uri invalid} -setup {} -body {
-        qc::is_uri_valid "http://test.co$%^&m"
+    test is-uri1.22 {is uri invalid} -setup {} -body {
+        qc::is uri "http://test.co$%^&m"
     } -result 0
 
-    test is_uri1.23 {is_uri invalid} -setup {} -body {
-        qc::is_uri_valid "http://test.com/$%^&m.pdf"
+    test is-uri1.23 {is uri invalid} -setup {} -body {
+        qc::is uri "http://test.com/$%^&m.pdf"
     } -result 0
 
-    test is_uri1.23 {is_uri invalid} -setup {} -body {
-        qc::is_uri_valid "///$%^&m.pdf"
+    test is-uri1.23 {is uri invalid} -setup {} -body {
+        qc::is uri "///$%^&m.pdf"
     } -result 0
 
+    test is-period-1.0 {qc::is period ""} {
```

Comment:
Tests written like:
```
test some-test {some description} {test_code} {result}
```

or 

```
test some-test {some description} -body {test_code} -result {result}
```

Don't think it will work if we mix the two by using the -result flag but not the -body flag